config/src/config/consensus_config.rs

```diff
@@ -197,7 +197,7 @@ impl Default for ConsensusConfig {
                 num_blocks_to_look_at: 12,
```

Comment:
increase this to 20, for less variance. that should still be only 2 seconds with increased blocks/s